code/modules/cargo/packs.dm

```diff
@@ -2637,6 +2650,26 @@
 	crate_type = /obj/structure/closet/crate/wooden
 	crate_name = "festive wrapping paper crate"
 
+/datum/supply_pack/misc/paper_work
+	name = "Freelance Paper work"
+	desc = "The Nanotrasen Primary Bureaucratic Database Intelligence (PDBI) reports that the station has not completed its funding and grant paperwork this solar cycle. In order to gain further funding, your station is required to fill out (10) ten of these forms or no additional capital will be disbursed. We have sent you ten copies of the following form and we expect every one to be up to Nanotrasen Standards." // Disbursement. It's not a typo, look it up.
+	cost = 700 // Net of 0 credits
```

Comment:
700 for a true net of 0, I thought crates were re-exported for 200 and then 200 for the paper work would be 400, turns out crates are worth 500 per export, meaning on top of the now 750 per crate and not 250, you could make 1050 on a 300 crafting tick crate. I think its not ok for it to be above 1000, for a basically free crate.